README.md

```diff
@@ -150,6 +148,14 @@ class Account < ApplicationRecord
 end
 ```
 
+Custom index can be handled with a `Arel::Nodes::SqlLiteral` instance:
+
+```ruby
+class Account < ApplicationRecord
+  upsert_keys upsert_keys Arel::Nodes::SqlLiteral.new('md5(my_long_field)')
```

Comment:
This is a typo, right? `upsert_keys upsert_keys`.

Reply:
Yes, rewrited